docs/management/index-lifecycle-policies/example-index-lifecycle-policy.asciidoc

```diff
@@ -1,23 +1,167 @@
 [role="xpack"]
+
 [[example-using-index-lifecycle-policy]]
-=== Example of using an index lifecycle policy
+=== Tutorial:  Use ILM to manage Filebeat time-based indices
```

Comment:
We maintain a list of attributes here: https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc

Try to use those when possible. They ensure we can make docs-wide term changes easily.
```suggestion
=== Tutorial:  Use {ilm-cap} to manage {filebeat} time-based indices
```